tests/unit/test_app.py

```diff
@@ -150,7 +169,7 @@ def test_post_image_create_endpoint_success(client, mock_data):
     assert res.json == build_result(
       res.json["url"].split("/")[-1],
       entry["real"],
-      entry["date"],
+      calculate_date(entry["date"]),
```

Comment:
This is a bandaid until my other PR is merged with a better setup for data in the mock FS